calico/felix/fiptables.py

```diff
@@ -612,30 +612,40 @@ def _calculate_ipt_modify_input(self):
         #
         # The chains are created if they don't exist.
         input_lines = []
-        affected_chains = self._txn.affected_chains
-        for chain in affected_chains:
-            if (self._grace_period_finished or
-                    chain not in self._chains_in_dataplane or
-                    chain not in self._txn.chains_to_stub_out):
-                # We're going to rewrite or delete this chain below, mark it
-                # for creation/flush.
-                input_lines.append(":%s -" % chain)
+        chains_to_flush = set()
```

Comment:
Can you rename this to chains_to_stub.  Flush is a bit confusing - is it different?  (If I'm wrong and it doesn't mean stub then please talk to me!)